pkgs/development/python-modules/nfstream/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, buildPythonApplication
+, psutil
+, numpy
+, dpkt
+, cffi
+, pandas
+}:
+
+buildPythonPackage rec {
+  pname = "nfstream";
+  version = "6.3.5";
+  format = "wheel";
+
+  src = fetchPypi {
+    inherit pname version;
+    format = "wheel";
+    sha256 = "1wqcz4vavay56gf8nn55s67wp61rmyw8s1mzki3b3zbbzwl1zbvy";
+    abi = "cp39";
+    python = "cp39";
+    platform = "manylinux1_x86_64";
+  };
+
+  propagatedBuildInputs = [ psutil numpy dpkt cffi pandas ];
```

Comment:
pythonImportsCheck and tests missing.

Reply:
adding 
`checkInputs = [ pytest ];`
is enough? What exactly pythonImportsCheck do?

Reply:
You want

```nix
pythonImportsCheck = [ "nfstream" ];
```
and

```nix
checkInputs = [ pytestCheckHook ];
```

and then fix/disable any failures.